identity-provider/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,171 @@
+// 5. The "edit info" and "unlink" buttons as well as the organization info will have to dissapear from the dashboard when the user presses 'unlink'.
+// Pop up alerts:
+// The "Your Information" alert will contained stored data from the settings page(whatever info is updated from pressing the "update profile" button.)
+// How to we show the dashbooard no organizations and the dashboard organizations page? Depends on what the if else statements say and what button the user presses?
+
+
+// Imports from material ui page, should they be within the DashboardOrganizations function or here at the top?
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+// ******************************************************
+
+function DashboardNoOrganizations() {
+
+    const voulenteerButton = () => {
+        console.log('This button will link to our relying parties website')
+    }
+    return (
+        <>
+        <h1>You're not currently connected to any organizations.</h1>
+        <p>Search for voulenteer oppertunities <button type = "submit" onClick = {voulenteerButton}> Here</button></p>
+        
+        </>
+    )
+}
+
+function DashboardOrganizations() {
+
+   // code copied from material ui: Will show the dialog component on the website
+  //  Will need a seperate one for each dialog's title and description
+  //  Will need another open, setOpen for the other dialog
+
+
+   const [open, setOpen] = React.useState(false);
+   const [dialogTitle] = React.useState('Hello World');
+    //  open is the current state value( in dialog, open = {open}), setOpen is the function that updates the state and we are setting it to true 
+  //  useState returns an array
+ 
+  // open is set to false. What does false do though? Maybe we can change the true and false statements to something else?
+  // if you were to call setOpen(true), it will change the value of open to true
+  // so how does this carry over to what we have going on so far?
+
+
+   const handleClickOpen = () => {
+     setOpen(true);
+   };
+ 
+   const handleClose = () => {
+     setOpen(false);
+   };
+ // *********************************************
+
+//  We want dialog to show up for each button, so do we need to put the return inside the buttons or keep it in the function?
+    const editInfoButton = () => {
+        console.log('This button will bring up a typed out alert which shows your information and contains x butttons to delete info with a cancel and save button.')
+    }
+
+    const unlinkButton = () => {
+        console.log('This button will bring up a typed alert which will ask you if you want to unlink from the organization the user pressed, with a cancel and confirm button.')
```

Comment:
This function will set the state variable for opening the Unlink Confirm Dialog component to true.
It would also set the state variable for which organization is being handled.